webrender/src/renderer.rs

```diff
@@ -412,6 +413,20 @@ pub struct Renderer {
     vr_compositor_handler: Arc<Mutex<Option<Box<VRCompositorHandler>>>>
 }
 
+#[derive(Debug)]
+pub enum InitError {
+    Shader(ShaderError),
+    Thread(std::io::Error),
+}
+
+impl std::convert::From<ShaderError> for InitError {
```

Comment:
I don't think you need the full path here: `std::convert::From` should be in prelude already